_includes/configuration/rtc_client_connected.md

```diff
@@ -0,0 +1,22 @@
+### `rtc_client_connected`
+
+This option allows applications to show when a user enters the RTC session. When used in combination with [`rtc_client_disconnected`](#rtc_client_disconnected), a user interface of connected users can be kept up to date.
+
+Only one `rtc_client_connected` event will be fired per client connection. Connecting to a session with multiple existing clients will fire separate `rtc_client_connected` events for each existing client. If a user connects using two or more editors (such as on desktop and on mobile), a separate `rtc_client_connected` event will be fired.
+
+To help with generating a user interface for connected users, an object for user details is provided by the [`rtc_user_details_provider`](#rtc_user_details_provider) option.
+
+{% if plugincode != "rtc" %}
+Required plugin
+: [Real-Time Collaboration (`rtc`)]({{site.baseurl}}/plugins/premium/rtc/)
+{% endif %}
+
+Type
+: Function (Promise)
```

Comment:
What's going on here? These are 2 different things so it's rather confusing.

Reply:
```suggestion
: Promise
```
?

Reply:
if this is better, let me know and I'll roll it out

Reply:
I'm not exactly sure what it's trying to say, so I can't say if that's correct. I wondered if it was `Function that returns a Promise`?

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/2091/commits/b3091a1ad3845121811a20d729af9663174b26ec